tox.ini

```diff
@@ -1,9 +1,10 @@
 [tox]
-envlist = py27,py33,py34,py35,pypy,pypy3
+envlist = py27,py33,py34,py35,py36,pypy,pypy3
 
 [testenv]
 deps=
     pytest>=2.2.3
     pypy,pypy3: psycopg2cffi>=2.6.1
     py27,py33,py34,py35: psycopg2>=2.4.6
+    py36: psycopg2>=2.4.6
```

Comment:
Actually, it appears that you own ci setup disagrees w/r/t pypy3, but that doesn't affect me or this pr